packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -820,6 +822,9 @@ class CupertinoTextField extends StatefulWidget {
   /// {@macro flutter.widgets.undoHistory.controller}
   final UndoHistoryController? undoController;
 
+  /// Determine whether [onTapOutside] can be called when is not focused.
```

Comment:
Consider changing to: `.....[onTapOutside] is called when this widget is not focused.`